molecule/default/molecule.yml

```diff
@@ -7,11 +7,17 @@ driver:
 platforms:
   - name: nexus-test
     image: ${DOCKER_BASE_IMAGE:-idealista/jdk:8u292-bullseye-adoptopenjdk-headless}
-    privileged: true
+    privileged: false
     capabilities:
       - SYS_ADMIN
+    tmpfs:
+      - /tmp
+      - /run
+      - /run/lock
     volumes:
       - '/sys/fs/cgroup:/sys/fs/cgroup:ro'
+    published_ports:
+      - 0.0.0.0:8081:8081/tcp
```

Comment:
I wouldn't publish any ports if we can avoid it :)

Reply:
Done